USAGE

```diff
@@ -0,0 +1,37 @@
+Pipeline for RecobundlesX tractography
+======================================
+
+USAGE
+
+nextflow run main.nf --root ... [OPTIONAL_ARGUMENTS]
+
+DESCRIPTION
+
+--root=/path/to/[root]              Root folder containing multiple subjects
+
+                                    [root]
+                                    ├── S1
+                                    │   ├── *anat.nii.gz
+                                    │   └── *tracking.trk
+                                    └── S2
+                                        └── *
+
+--atlas_config = Path to json config file
```

Comment:
These options need to be more explicit. I've never used rbx and I'm lost.

Reply:
Should I show an example in the USAGE ?
Should I provide a fake example in the repo (its like 5ko anyway)